module-bluetooth/Bluetooth/interface/profiles/HFP/HFP.cpp

```diff
@@ -361,7 +368,7 @@ namespace bluetooth
                                  hspSdpRecordHandle,
                                  rfcommChannelNr,
                                  agServiceName.data(),
-                                 0,
+                                 1,
```

Comment:
what does it stand for?

Reply:
It turns on ability to reject call by HFP, I added constexpr for it in [MOS-368].